pkgs/by-name/pa/passt/package.nix

```diff
@@ -14,6 +16,12 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-UK3Klpo3cp4EH42W16qh5WbZlCxb+ETQtWbpgatL/Dc=";
   };
 
+  postPatch = lib.optionalString (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    substituteInPlace Makefile --replace-fail \
+      'PAGE_SIZE=$(shell getconf PAGE_SIZE)' \
+      "PAGE_SIZE=$(${stdenv.hostPlatform.emulator buildPackages} ${lib.getExe getconf} PAGE_SIZE)"
+  '';
+
```

Comment:
We can do this unconditionally, when buildPlatform equals hostPlatform, emulator is simply the shell.

Reply:
✅ done